### PR TITLE
Outbox v2

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**
Farajaland PR: opencrvs/opencrvs-farajaland#348

### CODE REVIEW

**Comment by jpye-finch** [comment]
@tahmidrahman-dsi 
Just did a quick review. Looks great!

One thing is the messaging if a Registrar registers an application from In Progress and Validated. The messaging is Waiting to submit. Instead of Waiting to register.

Design:
image

**Comment by jpye-finch** [comment]
Also can you review the Mobile screens. 
image

Design:
https://www.figma.com/file/O6PevbBv0lApmXWYotR8sf/OpenCRVS-Design-Specifications-v2?node-id=7473%3A13078

**Comment by tahmidrahman-dsi** [comment]
> @tahmidrahman-dsi Just did a quick review. Looking great!
> 
> One thing is the messaging if a Registrar registers an application from In Progress and Validated. The messaging is Waiting to submit. Instead of Waiting to register. Sorry this wasn't documented as well as it could have been
> 
Thanks @jpye-finch , will amend the messages



> Also can you review the Mobile screens.
> 
> Design: https://www.figma.com/file/O6PevbBv0lApmXWYotR8sf/OpenCRVS-Design-Specifications-v2?node-id=7473%3A13078

Thanks! I am going to implement the mobile design

**Comment by jpye-finch** [comment]
@tahmidrahman-dsi @Zangetsu101 
Notion page to describe the copy instead of the Figma page:
https://www.notion.so/opencrvs/Outbox-c27ca926654646c4ae612efd19a02b7e

File: packages/client/src/views/OfficeHome/outbox/Outbox.tsx [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+        iconId = `failed${index}`
+        icon = () => <StatusWaiting id={iconId} key={iconId} />
+        statusText = formatMessage(waitingToRetry)
+        break
+      default:
+        // default act as  SUBMISSION_STATUS[SUBMISSION_STATUS.READY_TO_REGISTER]:
+        iconId = `waiting${index}`
+        icon = () => <StatusWaiting id={iconId} key={iconId} />
+        statusText = formatMessage(statusWaitingToRegister)
+    }
```
Comment (Zangetsu101) [inline packages/client/src/views/OfficeHome/outbox/Outbox.tsx:230]: @tahmidrahman-dsi Could you please refactor this part? Probably using a map would be easier to manage then this switch statement

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/views/OfficeHome/outbox/Outbox.tsx (hunk 1 above)
Comment (tahmidrahman-dsi) [reply packages/client/src/views/OfficeHome/outbox/Outbox.tsx:230]: Thanks very much, @Zangetsu101, will do.

**Comment by tahmidrahman-dsi** [comment]
disable reinstate button if offline 

image

**Review by Zangetsu101 — APPROVED** [review]

**Comment by Zangetsu101** [comment]
@tahmidrahman-dsi The e2e tests are failing when I tried to run them locally. Could you please take a look?

**Comment by tahmidrahman-dsi** [comment]
> @tahmidrahman-dsi The e2e tests are failing when I tried to run them locally. Could you please take a look?

Thanks @Zangetsu101 , I am on it

**Comment by Zangetsu101** [comment]
@tahmidrahman-dsi There are some conflicts in this PR

**Comment by tahmidrahman-dsi** [comment]
> @tahmidrahman-dsi There are some conflicts in this PR

Thanks @Zangetsu101 , resolved https://github.com/opencrvs/opencrvs-core/pull/3938/commits/0b304d6b74c399fda31a22f22a3a86845f0b0f6f

**Pull request merged**